src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url:
```

Comment:
Someday you should consider setting this up to have placeholders for the host/port, etc. within this property. That will make it easier, in the long run, for just those things to be overridden without having to duplicate the other pieces of the url. Here is an excessive example of that we used in RDW:
```
    url: "\
      jdbc:mysql://${spring.datasource.url-server:localhost:3306}/${spring.datasource.url-schema:warehouse}\
      ?useSSL=${spring.datasource.use-ssl:false}\
      &useLegacyDatetimeCode=${spring.datasource.use-legacy-datetime-code:false}\
      &characterEncoding=${spring.datasource.character-encoding:utf8}\
      &connectTimeout=${spring.datasource.connect-timeout:10000}\
      &socketTimeout=${spring.datasource.socket-timeout:40000}\
      "
```